tutorials/containers/introducing-microk8s/introducing-microk8s.md

```diff
@@ -0,0 +1,250 @@
+kube-system   pod/monitoring-influxdb-grafana-v4-8467db6558-7s7kq   2/2     Running   3          51m
+
+NAMESPACE     NAME                           TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)             AGE
+default       service/kubernetes             ClusterIP   10.152.183.1     <none>        443/TCP             53m
+default       service/microbot-service       NodePort    10.152.183.110   <none>        80:32648/TCP        12m
+kube-system   service/heapster               ClusterIP   10.152.183.89    <none>        80/TCP              51m
+kube-system   service/kube-dns               ClusterIP   10.152.183.10    <none>        53/UDP,53/TCP       51m
+kube-system   service/kubernetes-dashboard   ClusterIP   10.152.183.149   <none>        443/TCP             51m
+kube-system   service/monitoring-grafana     ClusterIP   10.152.183.54    <none>        80/TCP              51m
+kube-system   service/monitoring-influxdb    ClusterIP   10.152.183.163   <none>        8083/TCP,8086/TCP   51m
+
+NAMESPACE     NAME                                             DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+default       deployment.apps/microbot                         2         2         2            2           12m
+kube-system   deployment.apps/heapster-v1.5.2                  1         1         1            1           51m
+kube-system   deployment.apps/kube-dns                         1         1         1            1           51m
+kube-system   deployment.apps/kubernetes-dashboard             1         1         1            1           51m
+kube-system   deployment.apps/monitoring-influxdb-grafana-v4   1         1         1            1           51m
+
+NAMESPACE     NAME                                                        DESIRED   CURRENT   READY   AGE
+default       replicaset.apps/microbot-66df6f98d5                         2         2         2       12m
+kube-system   replicaset.apps/heapster-v1.5.2-5995dc5b8c                  1         1         1       51m
+kube-system   replicaset.apps/heapster-v1.5.2-7864d77c47                  0         0         0       51m
+kube-system   replicaset.apps/heapster-v1.5.2-dd4fd4c49                   0         0         0       51m
+kube-system   replicaset.apps/kube-dns-67b548dcff                         1         1         1       51m
+kube-system   replicaset.apps/kubernetes-dashboard-67d4c89764             1         1         1       51m
+kube-system   replicaset.apps/monitoring-influxdb-grafana-v4-8467db6558   1         1         1       51m
+```
+
+At the very top we have our microbot pods, `service/microbot-service` is the second in the services list. Our service has a ClusterIP through which we can access it. Notice however that our service is of type [NodePort][nodeport]. This means that our deployment is also available on a port on the host machine; that port is randomly selected and in this case it happens to be `32648`. All we need is to point our browser to http://localhost:32648
```

Comment:
*All we need to do is point...*

Full stop missing at the very end. Use backticks to be consistent with a previous URL.